docs/api.md

```diff
@@ -331,63 +393,178 @@ If the email address provided does not match any user in the system, that addres
 
-That said, if you do wish to delete the Field Key altogether, you can do so by issuing a `DELETE` request to its resource path. Field Keys cannot delete themselves.
+That said, if you do wish to delete the App User altogether, you can do so by issuing a `DELETE` request to its resource path. App Users cannot delete themselves.
 
 + Parameters
-    + id: `16` (number, required) - The numeric ID of the Field Key
+    + id: `16` (number, required) - The numeric ID of the App User
+
++ Response 200 (application/json)
+    + Attributes (Success)
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
+## Roles [/v1/roles]
+
+_(introduced: version 0.5)_
+
+The Roles API lists and describes each known Role within the system. Right now, Roles may not be created or customized via the API, but this will likely change in the future.
+
+Each Role contains information about the verbs it allows its assignees to perform. Some Roles have a system name associated with them; the Roles may always be referenced by this system name in request URLs, and system Roles are always read-only.
+
+### Listing all Roles [GET]
+
+Currently, there are no paging or filtering options, so listing `Role`s will get you every Role in the system, every time. There are no authorization restrictions upon this endpoint: anybody is allowed to list all Role information at any time.
+
++ Response 200 (application/json)
+    + Attributes (array[Role])
+
+### Getting Role Details [GET /v1/roles/{id}]
+
+Getting an individual Role does not reveal any additional information over listing all Roles. It is, however, useful for direct lookup of a specific role:
+
+The `id` parameter for Roles here and elsewhere will accept the numeric ID associated with that Role, _or_ a `system` name if there is one associated with the Role. Thus, you may request `/v1/roles/admin` on any ODK Central server and receive information about the Administrator role.
+
+As with Role listing, there are no authorization restrictions upon this endpoint: anybody is allowed to get information about any Role at any time.
+
++ Parameters
+    + id: `1` (string, required) - Typically the integer ID of the `Role`. You may also supply the Role `system` name if it has one.
+
++ Response 200 (application/json)
+    + Attributes (Role)
+
+## Assignments [/v1/assignments]
+
+_(introduced: version 0.5)_
+
+There are _two_ Assignments resources. This one, upon the API root (`/v1/assignments`), manages Role assignment to the entire system (eg if you are assigned a Role that gives you `form.create`, you may create a form anywhere on the entire server). The [other one](/reference/project-management/project-assignments), nested under Projects, manages Role assignment to that Project in particular.
```

Comment:
`eg` should probably be `e.g.`, but your usage is consistent, so this is your call.